ush/nems.configure.atm_aer.IN

```diff
@@ -0,0 +1,34 @@
+#############################################
```

Comment:
These are templates.
Why are they in `ush`
Shouldn't they be part of `parm` or a subdirectory within?

Reply:
aah yes.  I forgot about those spectral relics.

Reply:
Ignore this comment because we will be updating this branch with f/c-c and that may cause conflicts.